apps/widgets/src/pages/admin-dashboard.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import type { Widget, WidgetStatus } from '@walless/graphql';
+import { qlClient } from '@walless/graphql';
+import { updateWidgetStatus } from '@walless/graphql/mutation';
+import { allWidgets } from '@walless/graphql/query';
+import { Text } from '@walless/gui';
+import { HomeLayout } from 'components/layouts';
+import { WidgetInfo } from 'features/AdminDashboard';
+
+const fetchWidgetsData = async () => {
+	interface Data {
+		widgets: Widget[];
+	}
```

Comment:
don't re-declare interface on scoped context, just `never` for this case is enough, no one needed the Data interface anyway.